cabal-install/Distribution/Client/Setup.hs

```diff
@@ -1663,6 +1665,12 @@ installOptions showOrParseArgs =
           "Do not install anything, only print what would be installed."
           installDryRun (\v flags -> flags { installDryRun = v })
           trueArg
+
+      , option "" ["target-package-db"]
```

Comment:
Can we make this a hidden option?

Reply:
Can you give me a pointer where I can find a hidden option? I've tried looking for hiddenOption, and anything with hidden and option, but didn't turn anything up.

Reply:
@23Skidoo so I fail to figure out how to make this a hidden option. I also tried to set the help-text to `""` as per suggestion by @hvr, but that didn't drop it from being displayed either.